openquake/commonlib/readinput.py

```diff
@@ -228,6 +228,20 @@ def _update(params, items, base_path):
             params[key] = value
 
 
+def _warn_about_duplicates(cp):
+    sections = cp.sections()
+    prev_sections = []
+    for curr_sect in sections:
+        prev_sections.append(curr_sect)
+        other_sects = [sect for sect in sections if sect not in prev_sections]
```

Comment:
This algorithm is quadratic and really wrong to use in general (one should use sets) however configuration files has extremely few sections so it is a non-issue here.